pkg/chartvalues/apiextensions_app_e2e_template.go

```diff
@@ -1,42 +1,55 @@
 package chartvalues
 
 const apiExtensionsAppE2ETemplate = `
-app:
-  name: "{{ .App.Name }}"
-  namespace: "{{ .App.Namespace }}"
-  catalog: "{{ .App.Catalog }}"
-  config:
-    configMap:
-      name: "{{ .App.Config.ConfigMap.Name }}"
-      namespace: "{{ .App.Config.ConfigMap.Namespace }}"
-    secret:
-      name: "{{ .App.Config.Secret.Name }}"
-      namespace: "{{ .App.Config.Secret.Namespace }}"
-  kubeConfig:
-    inCluster: {{ .App.KubeConfig.InCluster }}
-    secret:
-      name: "{{ .App.KubeConfig.Secret.Name }}"
-      namespace: "{{ .App.KubeConfig.Secret.Namespace }}"
-  version: "{{ .App.Version }}"
+apps:
+  - name: "{{ .App.Name }}"
+    namespace: "{{ .App.Namespace }}"
+    catalog: "{{ .App.Catalog }}"
+    config:
+      configMap:
+        name: "{{ .App.Config.ConfigMap.Name }}"
+        namespace: "{{ .App.Config.ConfigMap.Namespace }}"
+      secret:
+        name: "{{ .App.Config.Secret.Name }}"
+        namespace: "{{ .App.Config.Secret.Namespace }}"
+    kubeConfig:
+      inCluster: {{ .App.KubeConfig.InCluster }}
+      secret:
+        name: "{{ .App.KubeConfig.Secret.Name }}"
+        namespace: "{{ .App.KubeConfig.Secret.Namespace }}"
+    version: "{{ .App.Version }}"
+  - name: "chart-operator"
```

Comment:
This makes sense as its the app CR for bootstrapping chart-operator. The other app CR is configurable.

But can we have a comment here explaining?

Reply:
Sure, that could be easier for someone to read this later.

Reply:
Damn too late but can we make it `Added app CR for bootstrapping chart-operator`?

As the repo is e2etemplates so this is all for e2e tests. 😄